app/src/components/common/outcomes/index.tsx

```diff
@@ -96,8 +95,11 @@ const Outcomes = (props: Props) => {
   const messageErrorToRender = () => {
     return (
       <>
-        <ErrorStyled>{messageProbabilitiesError}</ErrorStyled>
-        <ErrorStyled>{messageEmptyNamesError}</ErrorStyled>
+        {errorMessages.map((errorMessage, index) => (
```

Comment:
If I'm getting this right, `errorMessages` will be `['', '']` (an array with two empty strings) when everything is correct. So these two components will be rendered, albeit invisible. I'm fine with this, just making sure I'm understanding it.

Reply:
Mmm yes, you getting this right, but know that you say this, I don't like it, so I change the mechanim, now is not render empty strings anymore.